lib/ember-test-helpers/test-module.js

```diff
@@ -230,6 +230,13 @@ export default Klass.extend({
     this.container = items.container;
     this.registry = items.registry;
 
+
+    var viewRegistry = this.container.lookup('-view-registry:main');
+    if (viewRegistry) {
+      var View = Ember.__loader.require('ember-views/views/view').default;
+      View.views = viewRegistry;
```

Comment:
This gets weirder. Even if I use lookupFactory, `_viewRegistry` is only injected onto things registered as `view:foo`, not things registered as `component:foo`. That seems like a potential Ember bug, although it doesn't effect a normal app since they use `view:toplevel` (not `component:toplevel`, and then each view inherits a viewRegistry from its parent.